src/parameterStore.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import AWS from 'aws-sdk';
+
+const { IS_OFFLINE, AWS_REGION, STAGE } = process.env;
+
+function getSSM() {
+    if (IS_OFFLINE === 'true') {
+        return new AWS.SSM({
+            region: 'us-west-2',
+            accessKeyId: process.env.ACCESS_KEY,
+            secretAccessKey: process.env.SECRET_KEY,
+        });
+    }
+
+    return new AWS.SSM();
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export async function getIntegrationTransformUrl(): Promise<string> {
+    const ssm = getSSM();
+    let path = process.env.INTEGRATION_TRANSFORM_PATH;
+    if (path === undefined) {
+        throw new Error('INTEGRATION_TRANSFORM_PATH is not defined in environment variables');
+    }
+    if (IS_OFFLINE === 'true') {
+        path = `fhir-service.integration-transform.${AWS_REGION}.${STAGE}.url`;
+    }
```

Comment:
Is the order of If conditions right? This looks like it will always fail in offline mode

Reply:
`serverless offline` renders the environment variable as `[object Object]` so technically it's not rendered as `undefined`. The code still works correctly.

This code catches case in which someone edits `serverless.yaml` and deletes the environment variable for `INTEGRATION_TRANSFORM_PATH`